src/FizzBuzz.java

```diff
@@ -1,17 +1,25 @@
+import java.util.ArrayList;
+
 public class FizzBuzz {
     public static void main(String[] args) throws Exception {
 
+        ArrayList<String> result = new ArrayList<String>();
+
         for (int i = 1; i <= 100; i++) {
 
-            if ( i % 3 == 0 && i % 5 == 0) {
-                System.out.println("FizzBuzz");
-            } else if (i % 3 == 0) {
-                System.out.println("Fizz");
-            } else if (i % 5 == 0) {
-                System.out.println("Buzz");
-            } else {
-                System.out.println(i);
+            if (i % 3 == 0) {
+                result.add("Fizz");
+            }
+            if (i % 5 == 0) {
+                result.add("Buzz");
             }
+            if (result.size() == 0) {
+                result.add(String.valueOf(i));
+            }

```

Comment:
This is also a scenario in which I can introduce my favourite programming operator called the "ternary operator" https://www.w3schools.com/java/java_conditions_shorthand.asp

its perfect for when you want to do "if <condition> then <result> else <other result>". In this case it could be within the `System.out.println` you can check the length of the array/stringBuilder and return `i` if its 0, or the join if not